### ISSUE TITLE
Exported GIFs not looping in browsers

### ISSUE THREAD

**Issue by levrik** [wulkano/kap#502]
**macOS Version:** 10.13.6
**Kap Version:** 3.0.0 (3.0.0.918)

#### Steps to reproduce
Record something and export as GIF

#### Current behaviour
Doesn't loop in Chrome and Firefox but in macOS quick look an Preview.app

#### Expected behaviour
It loops in browsers

**Comment by karaggeorge** [comment]
Hey @levrik can you make sure the `Loop exports` option is toggled in the settings?

![screenshot 2018-09-19 07 16 31](https://user-images.githubusercontent.com/8822835/45750172-1c7db600-bbdc-11e8-8a24-a46e725e4b7d.png)

**Comment by levrik** [comment]
Yes. It's checked.
![image](https://user-images.githubusercontent.com/9491603/45875305-88584e00-bd97-11e8-9875-9cd3b1a930fb.png)
As I said. It's also looping outside browsers.

**Comment by skllcrn** [comment]
Could it be that some thing like `<img loop="">` is required when displayed in a browser? Don't know why it wouldn't get that information from the GIF itself, though.

**Comment by skllcrn** [comment]
Scratch that. Even though we don't have a setting to toggle looping in Kap 2.0, GIFs exported do loop by default and work in browsers.

**Comment by levrik** [comment]
Are you doing something different with GIF exports in 3.0 ?

**Comment by karaggeorge** [comment]
Not really. We kept everything the same. Also if the gifs are looping locally, that means looping is on. So it might be something around the way the gifs are used in the browser. How did you use them in the browser and which browser? I can try to investigate a bit more

**Comment by levrik** [comment]
Hey @karaggeorge. I used them in latest Google Chrome version on macOS pasted to GitHub. They stopped looping when I upgraded to Kap 3.x

**Comment by albinekb** [comment]
![kapture 2018-09-24 at 22 24 14](https://user-images.githubusercontent.com/5027156/45984581-ffc10280-c061-11e8-8a97-606d767421ac.gif)

This is recorded with Kap beta release 1.0, it loops here on github. What plugin did you use for export? @levrik

**Comment by levrik** [comment]
@albinekb 	Plugin?! I just clicked on GIF in the export window.

**Comment by albinekb** [comment]
Then that's the default share service, it should work. Can you try downloading the latest release? https://github.com/wulkano/kap/releases/download/v3.0.0-beta.1/kap-beta-3.0.0-beta.1.dmg

This is recorded with that release, with looping turned on in settings:
![kapture 2018-09-25 at 18 52 53](https://user-images.githubusercontent.com/5027156/46029629-43644c80-c0f4-11e8-9135-c98fbd27ae18.gif)
